WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/ui/TimeSynchronityChecker.java

```diff
@@ -66,16 +64,19 @@ public void run() {
 				if (!deltaIsWithinReasonableBoundaries) {
 					WatchDogLogger.getInstance().logInfo(
 							"System suspend detected!");
-					Perspective openedPerspective = intervalManager
-							.getInterval(PerspectiveInterval.class)
-							.getPerspectiveType();
 					intervalManager.closeAllIntervals(new Date(
 							previousExecutionDate + UPDATE_RATE));
 					intervalManager.generateAndSetSessionSeed();
 					eventManager.update(new WatchDogEvent(this,
-							EventType.START_IDE));
-					eventManager.update(new WatchDogEvent(openedPerspective,
+							EventType.START_IDE));					
+
```

Comment:
Hm, I am not sure I like this change because this part of the functionality should be on the client side, not the core (have a sub classes of TimeSynchronityChecker?).